### ISSUE TITLE
Provide option to unquote localparts when normalising

### ISSUE THREAD

**Issue by tdelaney-leadiro** [RohanNagar/jmail#168]
If a quoted localpart only contains a string that would be valid as an unquoted localpart, it would be useful to have an option to have normalisation return the localpart unquoted e.g. with the option enabled:

`"test.1"@example.org` would normalise to `test.1@example.org`
`"test..1"@example.org` would remain as `"test..1"@example.org` (because consecutive `.` can only appear in a quoted localpart)

The use case for this is detecting duplicate email addresses.

**Comment by RohanNagar** [comment]
Hey @tdelaney-leadiro, thanks for opening this issue! This seems like a great enhancement. I will try to get to this as soon as I can.

**Comment by RohanNagar** [comment]
This option is now available in v1.6.0! There are two ways to configure it:

1. Set the `-Djmail.normalize.strip.quotes=true` JVM system property.
2. Call the new `normalize(boolean)` method and pass in `true`.

Example:

```java
Email myEmail = Email.of("\"test.1\"@example.org").get();

String normalized = myEmail.normalize(true);

// normalized == "test.1@example.org"
```
